fa20/syllabus.yml

```diff
@@ -2,17 +2,18 @@
   required:
     id: 8f863b4f2ad5d2600cb549cf885be1065f9423efbd7526a3f3e73457fab26bb7
     date: !Timestamp 2020-09-15 18:00:00
-    title: meeting00
+    title: a-few-useful-things
     authors: ["PaulLunis]
-    filename: meeting00
+    filename: useful-ml
     cover-image: 'https://cacm.acm.org/system/assets/0000/9220/092012_CACMpg79_A-Few-Useful-Things2.large.jpg?1476779416&1348149730'
-    tags: ["ai","machine learning"]
+    tags: ["ai","machine learning","review paper","data science","ml research"]
```

Comment:
Good, it is an intro paper, so maybe that's worth denoting in the tags? (There's very little that the tags _shouldn't_ touch on (short of just pasting the entire paper. 😂 )